src/nets/e2e_asr_attctc.py

```diff
@@ -606,7 +619,13 @@ def recognize_beam(self, h, recog_args, char_list):
                 hyp['a_prev'] = att_w
 
                 # get nbest local scores and their ids
-                local_scores = F.log_softmax(self.output(z_list[-1])).data
+                if rnnlm:
+                    rnnlm_state, z_rnnlm = rnnlm.predictor(hyp['rnnlm_prev'], hyp['yseq'][i])
+                    hyp['rnnlm_prev'] = rnnlm_state
```

Comment:
Is this necessary? hyp['rnnlm_prev'] maybe not used anymore.

Reply:
Thanks! Maybe You're right. We don't need `hyp[...] = ...`. I'll test it.